packages/instantsearch.css/src/themes/algolia.scss

```diff
@@ -126,6 +127,49 @@ a[class^='ais-'] {
   display: none;
 }
 
+@media (hover: none) and (pointer: coarse) {
+  .ais-Carousel-navigation {
+    display: none;
+  }
+}
+
+.ais-Carousel-navigation,
+.ais-Carousel-navigation[hidden] {
+  opacity: 0;
+  visibility: hidden;
+}
+
+.ais-Carousel:hover .ais-Carousel-navigation:not([hidden]) {
+  opacity: 1;
+  visibility: visible;
+}
+
+.ais-Carousel-list {
+  padding-bottom: 0.5rem;
+  gap: 0.5rem;
+  outline-color: $algolia-blue;
+  outline-offset: 0.5rem;
+  scrollbar-color: $port-gore $ghost;
+  scrollbar-width: thin;
+}
+
+.ais-Carousel-list::-webkit-scrollbar {
+  height: 2px;
+}
+
+.ais-Carousel-list::-webkit-scrollbar-track {
+  background-color: $ghost;
+}
+
+.ais-Carousel-list::-webkit-scrollbar-thumb {
+  background-color: $port-gore;
+}
+
+.ais-Carousel-item * {
+  outline-color: $algolia-blue;
+  outline-offset: 3px;
+}
```

Comment:
I missed it in my own task, but if I'm not wrong this rule does nothing as it lacks style/width.

We can safely remove it (same for satellite).